src/hooks/useIsProfileValidated.ts

```diff
@@ -3,15 +3,16 @@ import isBoolean from 'lodash/isBoolean'
 import isEthereumAddress from 'validator/lib/isEthereumAddress'
 
 import { Governance } from '../clients/Governance'
+import { AccountType } from '../entities/User/types'
 
 import { DEFAULT_QUERY_STALE_TIME } from './constants'
 
-function useIsProfileValidated(address: string | null) {
+function useIsProfileValidated(address: string | null, accounts: AccountType[]) {
```

Comment:
Default could be `USER_FULLY_VALIDATED` here so you don't have to add it everywhere.